Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -179,7 +179,7 @@ void ABenchmarkGymGameMode::BuildExpectedActorCounts()
 
 	const int32 TotalDropCubes = TotalNPCs + ExpectedPlayers;
 	const int32 DropCubeCountVariance = FMath::CeilToInt(TotalDropCubes * 0.1f) + 2;
-	AddExpectedActorCount(ExpectedDropCubeCount, DropCubeClass, TotalDropCubes, DropCubeCountVariance);
+	AddExpectedActorCount(DropCubeClass, TotalDropCubes - DropCubeCountVariance, TotalDropCubes + DropCubeCountVariance);
```

Comment:
Should this be +- DropCubeCountVariance*0.5?

Reply:
Yes and no. I think the actual variance is good here. Perhaps DropCubeCountVariance  should be rename for clarity though. Maybe just HalfDropCubeCountVariance? What do you think?

Reply:
Sounds fine, think I just misunderstood the previous usage!